sql/core/src/test/java/test/org/apache/spark/sql/JavaBeanDeserializationSuite.java

```diff
@@ -115,6 +124,70 @@ public void testBeanWithMapFieldsDeserialization() {
     Assert.assertEquals(records, MAP_RECORDS);
   }
 
+  private static final List<Row> ROWS_SPARK_22000 = new ArrayList<>();
```

Comment:
All the static initialization could just be in a static block rather than split it up, but could this all be local to the new test method if that's the only place it's used?

Reply:
I just followed the approach the test class already has, but I agree they can be local to new test method. Will address.